ios/RNGestureHandlerPointerTracker.m

```diff
@@ -100,7 +117,9 @@ - (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event
     data[i] = [self extractPointerData:index forTouch:touch];
   }
   
-  _pointerData = [[NSArray alloc] initWithObjects:data count:[touches count]];
+  _changedPointersData = [[NSArray alloc] initWithObjects:data count:[touches count]];
+  // extract all touches lasr to include the ones that were just added
```

Comment:
A typo.
```suggestion
  // extract all touches last to include the ones that were just added
```

Reply:
Fixed in https://github.com/software-mansion/react-native-gesture-handler/pull/1732/commits/108bba6708f5f4468eb9010ea013ed159ce23854